### PR TITLE
New Float Field

### PR DESCRIPTION
**Pull request by Margaferrez**
Changes:
- Add float Field with precision
- Add float min and max validation

### CODE REVIEW

File: kim/field.py [hunk 1]
```diff
@@ -442,6 +442,49 @@ class UserMapper(Mapper):
     marshal_pipeline = DecimalMarshalPipeline
     serialize_pipeline = DecimalSerializePipeline
 
+class FloatFieldOpts(FieldOpts):
```
Comment (Margaferrez) [inline kim/field.py]: I was thinking on inherit from Decimal.. I'm happy to do a more reusable structure if you want me to.

Comment (jackqu7) [inline kim/field.py]: I think actually Decimal should inherit from Float. Could you do it that way round? Otherwise this looks great, thanks.

Comment (Margaferrez) [inline kim/field.py]: Yes, I'll do it ASAP, thanks :)

**Comment by jackqu7** [comment]
Awesome, thanks!

**Pull request merged**